docs/src/index.md

```diff
@@ -1,72 +1,96 @@
 # ChainRules
 
-[ChainRules](https://github.com/JuliaDiff/ChainRules.jl) provides a variety of common utilities that can be used by downstream [automatic differentiation (AD)](https://en.wikipedia.org/wiki/Automatic_differentiation) tools to define and execute forward-, reverse-, and mixed-mode primitives.
+[Automatic differentiation (AD)](https://en.wikipedia.org/wiki/Automatic_differentiation) is a set of techniques for obtaining derivatives of arbitrary functions.
+There is a surprisingly large number of packages for doing AD in Julia.
+ChainRules isn't one of these packageis.
 
-## Introduction
+The AD packages essentially combine derivatives of simple functions into derivatives of more complicated functions.
+They differ in the way they combine the derivatives, but they all require a common set of derivatives of simple functions (primitives/rules).
```

Comment:
the difference is less  in how they combine derivatives and more in how they break down to the simple functions, IMO.
But that might be too much detail.